src/components/header/CustomHeader.tsx

```diff
@@ -6,12 +6,14 @@ interface Props {
   left?: React.ReactNode
   right?: React.ReactNode
   title?: React.ReactNode | string
+  subTitle?: string | React.ReactNode
```

Comment:
remove?

Reply:
Removed in b93f45c73831ae54a9235201fcb627d7b502d89c!